crates/rpc/rpc/src/debug.rs

```diff
@@ -1143,8 +1143,29 @@ where
         Ok(())
     }
 
-    async fn debug_db_get(&self, _key: String) -> RpcResult<()> {
-        Ok(())
+    async fn debug_db_get(&self, key: String) -> RpcResult<Option<Bytes>> {
+        // According to Geth impl, bytecode keys are: 0x63 (prefix) + 32 bytes (code_hash)
+        // https://github.com/ethereum/go-ethereum/blob/737ffd1bf0cbee378d0111a5b17ae4724fb2216c/core/rawdb/schema.go#L120
+        let key_bytes =
+            decode(&key).map_err(|_| EthApiError::InvalidParams("Invalid hex key".to_string()))?;
+        if key_bytes.len() != 33 {
+            return Err(EthApiError::InvalidParams(format!(
+                "Key must be 33 bytes, got {}",
+                key_bytes.len()
+            ))
+            .into());
+        }
+        if key_bytes[0] != 0x63 {
+            return Err(EthApiError::InvalidParams(
+                "Key prefix must be 0x63 for bytecode".to_string(),
+            )
+            .into());
+        }
+
+        let code_hash = B256::from_slice(&key_bytes[1..33]);
```

Comment:
this endpoint supports more lookups that we perhaps want to add eventually

I think we can document this here, for now just support code by hash is fine